src/main/java/hudson/plugins/collabnet/orchestrate/BuildNotifier.java

```diff
@@ -154,44 +155,39 @@ public OptionalAssociationView(ConnectionFactory connectionFactory) {
         }
     }
 
-    public static class OptionalEventQ{
+    public static class RadioConfig{
 
-        private final String serverUrl;
-        private final String serverUsername;
-        private final Secret serverPassword;
-        private final String sourceKey;
+        private String serverUrl;
```

Comment:
Why are there multiple url/Username/Password fields? Since you have combined OptionalEventQ and OptionalWebhook into RadioConfig, Can't you just have `url`, `username`, `password` fields, and use them for eventq and webhook?

Reply:
Maintaining a different set of variables for both eventq and webhook with minimal changes, which helps in easy processing. As we going to deprecate eventq in the future release, the eventq related code will be removed completely.

Reply:
That would not be the reason!. However, it is ok to keep them separate so that the plugin continues to support existing configurations